WMF Framework/Wikipedia.xcdatamodeld/.xccurrentversion

```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>_XCCurrentVersionName</key>
-	<string>Wikipedia 3.xcdatamodel</string>
+	<string>Wikipedia 4.xcdatamodel</string>
```

Comment:
@joewalsh Just curious - why the new version for this data model but not the other ones?

Reply:
Good question, and there's not a good reason. Core Data will generate the mapping automatically even if the version doesn't change, but it's probably better to be explicit about it whenever we change a model. I've added new versions for the other models and reverted the base versions in 2e33e9f.